From 13042d1c3b32e78ac2cf89298362a97d234b7524 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Mon, 27 Nov 2017 10:41:37 +0800
Subject: [PATCH] Revert "ARM: dts: rk3126-bnd-d708: set VDD_LOG to 0.9V when
 suspend"

We had better follow develop-3.10 for stability.

This reverts commit 1b16fb98142c4d6ac57abe2fe6ce58c521e90c10.

Change-Id: Ic576675d069c02d15906463d062c44ce6e8d5195
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk3126-bnd-d708.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3126-bnd-d708.dts b/arch/arm/boot/dts/rk3126-bnd-d708.dts
index 366b594adae6..0fdd059d5bad 100644
--- a/arch/arm/boot/dts/rk3126-bnd-d708.dts
+++ b/arch/arm/boot/dts/rk3126-bnd-d708.dts
@@ -396,7 +396,7 @@
 				regulator-boot-on;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <900000>;
+					regulator-suspend-microvolt = <1000000>;
 				};
 			};
 
-- 
2.35.3

